warehouse/migrations/versions/7f0d1b5af8c7_add_tables_for_storing_ses_information.py

```diff
@@ -0,0 +1,154 @@
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Add tables for storing SES information
+
+Revision ID: 7f0d1b5af8c7
+Revises: 6418f7d86a4b
+Create Date: 2018-04-02 15:29:59.499569
+"""
+
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import postgresql
+
+revision = "7f0d1b5af8c7"
+down_revision = "6418f7d86a4b"
+
+
+SESEmailStatuses = sa.Enum(
+    "Accepted", "Delivered", "Soft Bounced", "Bounced", "Complained",
+    name="ses_email_statuses",
+)
+
+
+SESEventTypes = sa.Enum(
+    "Delivery", "Bounce", "Complaint",
+    name="ses_event_types",
+)
+
+
+def upgrade():
+    op.create_table(
+        "ses_emails",
+        sa.Column(
+            "id",
+            postgresql.UUID(as_uuid=True),
+            server_default=sa.text("gen_random_uuid()"),
+            nullable=False,
+        ),
+        sa.Column(
+            "created",
+            sa.DateTime(),
+            server_default=sa.text("now()"),
+            nullable=False,
+        ),
+        sa.Column(
+            "status",
+            SESEmailStatuses,
+            nullable=False,
+            server_default="Accepted",
+        ),
+        sa.Column("message_id", sa.Text(), nullable=False),
+        sa.Column("from", sa.Text(), nullable=False),
+        sa.Column("to", sa.Text(), nullable=False),
```

Comment:
should we index this to help with admin search?

Reply:
I've added an index now!